### PR TITLE
Fix question review on learn paths

### LINKED ISSUE #2672: [Learning Paths Quiz] Allow users to review answers
**Description**

https://user-images.githubusercontent.com/85789734/159272756-0d4a1d92-0938-4fc0-ac95-c5b350843faf.mov

We immediately move to the next question on clicking on any option. The quiz auto-submits on answering the last question.

**Expected Behavior**

Allow the user to review their answers before submitting. 

- On answering one question, the selected option should remain highlighted and the user should be moved to the next question only on clicking "Next" and not immediately.
-  Similarly, on answering the last question, the quiz should not auto-submit. User might want to visit a previous question, and so the option to "Submit" should specifically be available.

**Environment:**
- Host OS: 
- Browser: 

---

[Optional] **Additional Context**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Lezette**
Signed-off-by: elizabeth <leezeelola@gmail.com>

**Description**

This PR fixes #2672

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 3407585c98f0ece575be38b7aa19be1f51efeb7a at: https://624313fb1923eb143f6432ea--layer5ng.netlify.app

**Review by debo19 — COMMENTED** [review]
@Lezette We should have the selected option remain highlighted before moving to the next question

**Comment by leecalcote** [comment]
Thank you, @Lezette!

**Comment by subhradwip01** [comment]
> @Lezette We should have the selected option remain highlighted before moving to the next question

Ya..I am working on that problem #2671

**Comment by l5io** [comment]
🚀 Preview for commit a492dbe2ceea217d568aeb1d670e508ab51b2628 at: https://624888e2eba34b43a06252e7--layer5ng.netlify.app

**Comment by subhradwip01** [comment]
> rocket Preview for commit [a492dbe](https://github.com/layer5io/layer5/commit/a492dbe2ceea217d568aeb1d670e508ab51b2628) at: https://624888e2eba34b43a06252e7--layer5ng.netlify.app

Actually, it is not storing the answer .. so whenever we move to the next question it got deselected.. Don't worry about this problem it is solved in this PR #2697..you can check it out there

**Comment by Lezette** [comment]
@debo19 I've fixed the review part while there's another PR that fixes the highlight issue [here](https://github.com/layer5io/layer5/pull/2697)

**Comment by debo19** [comment]
@Lezette @subhradwip01 lets discuss both your PRs in the websites call today

**Comment by subhradwip01** [comment]
> 

👍

File: src/components/Learn-Components/Pagination/index.js [hunk 1]
```diff
@@ -51,14 +51,14 @@ const Pagination = ({ TOCData, chapterData, location, showQuizModal }) => {
...
-      //         </QuizWrapper>
-      //   )
+      // : null
+      : (
+        <QuizWrapper onClick={showQuizModal}>
+          <Button secondary title="Take Quiz"
+            external={false} 
+          />
+        </QuizWrapper>
+      )
```
Comment (Nikhil-Ladha) [inline src/components/Learn-Components/Pagination/index.js:61]: The fix is working, let's get it back commented.

Comment (Lezette) [reply src/components/Learn-Components/Pagination/index.js]: done @Nikhil-Ladha

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**